system-addon/lib/ActivityStreamMessageChannel.jsm

```diff
@@ -128,33 +128,37 @@ this.ActivityStreamMessageChannel = class ActivityStreamMessageChannel {
   }
 
   /**
-   * sendToPreloaded - Sends an action to the preloaded browser, if any
+   * sendToPreloaded - Sends an action to each preloaded browser, if any
    *
    * @param  {obj} action A redux action
    */
   sendToPreloaded(action) {
-    const preloadedBrowser = this.getPreloadedBrowser();
-    if (preloadedBrowser && action.data) {
-      try {
-        preloadedBrowser.sendAsyncMessage(this.outgoingMessageName, action);
-      } catch (e) {
-        // The preloaded page has already been consumed, so just ignore.
+    const preloadedBrowsers = this.getPreloadedBrowser();
+    if (preloadedBrowsers && action.data) {
+      for (let preloadedBrowser of preloadedBrowsers) {
+        try {
+          preloadedBrowser.sendAsyncMessage(this.outgoingMessageName, action);
+        } catch (e) {
+          // The preloaded page has already been consumed, so just ignore.
```

Comment:
This actually isn't true, can you update to say "is no longer available" or something